server/weaverbird/steps/aggregate.py

```diff
@@ -0,0 +1,47 @@
+from typing import Iterator, List, Literal, Tuple, Union
+
+import pandas as pd
+from mypy.applytype import Optional
+from pandas import DataFrame
+from pydantic import Field
+from pydantic.main import BaseModel
+
+from weaverbird.steps.base import BaseStep
+
+ColumnName = Union[str, int, float]
+AggregateFn = Literal['avg', 'sum', 'min', 'max']
```

Comment:
Careful here: we will need a mapping (e.g. `avg` in the weaverbird language => `mean` in pandas language)